src/components/EventDetails/EventDetails.js

```diff
@@ -120,10 +127,12 @@ import { switchAccountAction } from 'actions/accountsActions';
 import { goToInvitationFlowAction } from 'actions/referralsActions';
 import { updateTransactionStatusAction } from 'actions/historyActions';
 import { lookupAddressAction } from 'actions/ensRegistryActions';
+import { setActiveBlockchainNetworkAction } from 'actions/blockchainNetworkActions';
```

Comment:
We won't be using this anymore, please use if asset.symbol === BTC condition to determine BTC specific parts

Reply:
As of this current version, we still use networks. So I'm keeping this for now. Will adjust when your PR will be in and this branch will be updated. To be sure that it works :)